composer.json

```diff
@@ -25,12 +25,12 @@
     },
     "require-dev": {
         "symfony-cmf/testing": "^1.3",
-        "matthiasnoback/symfony-dependency-injection-test": "~0.7,>=0.7.4",
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.4",
         "matthiasnoback/symfony-config-test": "^1.2.3",
-        "sonata-project/block-bundle": ">=2.2.8",
-        "sonata-project/core-bundle": ">=2.2.3",
+        "sonata-project/block-bundle": "^2.2.8",
+        "sonata-project/core-bundle": "^2.2.3",
         "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": ">=1.0.0-RC1",
+        "symfony-cmf/tree-browser-bundle": "^1.0.0|^2.0.0",
```

Comment:
Why would we have a conflict when we don't use 2.0? Afaik, nothing requires us to use 2.0 now.

I think it's in here because we didn't run the lowest dep test with stable deps, so we had to make sure at least RC1 was released (because some older version had issues). We do now use stable deps, so it can indeed be removed.